github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -941,6 +941,90 @@ presubmits:
           hostPath:
             path: /dev/vfio/
             type: Directory
+
+  - name: pull-kubevirt-e2e-kind-1.19-sriov
+    skip_branches:
+      - release-\d+\.\d+
+    annotations:
+      fork-per-release: "true"
+      k8s.v1.cni.cncf.io/networks: 'multus-cni-ns/sriov-passthrough-cni,multus-cni-ns/sriov-passthrough-cni'
+      testgrid-dashboards: kubevirt-presubmits
+    always_run: true
```

Comment:
This should probably be false while we don't have it in kubevirt/kubevirt.

Reply:
Absolutely, I missed this setting, thanks for raising it! These jobs should replace the existing 1.17 lanes.